code/modules/mob/living/carbon/human/species.dm

```diff
@@ -250,8 +250,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
   */
 /datum/species/proc/qualifies_for_rank(rank, list/features)
 	if(rank in GLOB.command_positions)
-		return 0
-	return 1
+		return TRUE
+	return FALSE
```

Comment:
This proc isn't actually used anywhere, and its return values were reversed for such a long time(5 years), might be better to just delete all instances of it